lib/language/messages_en.py

```diff
@@ -149,6 +149,12 @@ def all_messages():
             "all_usernames": "the usernames",
             "all_passwords": "the passwords",
             "timeout_seconds": "the timeout seconds",
+            "Invalid_shodan_api_key": "{0}",
+            "shodan_api_key": "Shodan api key for shodan recon",
+            "shodan_results_found": "Found results: {0}",
+            "shodan_results_not_found": "Didn't found anything in the shodan database",
```

Comment:
typo/grammar fix, please change to : "Didn't find anything in the Shodan database"